### PR TITLE
Create Vere profiling grant

### PR DESCRIPTION
**Pull request by mcevoypeter**
@thelifeandtimes and @joemfb, please make any changes you deem necessary by pushing directly to this branch. There are a few bits of information that I'm not able to fill in, namely the `reward`, `grant_id`, and `work_request_link` metadata fields and the compensation in the `Duration and Compensation` section. These missing bits are clearly marked with `TODO` in the draft.

### CODE REVIEW

**Review by thelifeandtimes — APPROVED** [review]
lgtm, will defer to @joemfb if there are any last details to add.

**Review by joemfb — APPROVED** [review]
LGTM

**Pull request merged**
